source/_integrations/media_source.markdown

```diff
@@ -48,10 +48,11 @@ This example makes two different folders available to the integration:
 
 ```yaml
 # Example configuration.yaml
-homeassistant:
-  media_dirs:
-    local: /media
-    recording: /mnt/recordings
+homeassistant
+  media_source:
+    media_dirs:
+      local: /media
+      recording: /mnt/recordings
```

Comment:
This is incorrect.

A) the YAML is now invalid YAML.
B) The configuration option is named `media_dirs` and is a core configuration option. `media_source` is not.